src/table/cells/handleKeyPress.js

```diff
@@ -0,0 +1,74 @@
+const moveFocus = (tableRows, nextRow, nextCol) => {
+  const nextCell = tableRows[nextRow].getElementsByClassName('sn-table-cell')[nextCol];
+  nextCell.focus();
+  nextCell.setAttribute('tabIndex', '0');
+};
+
+const arrowKeysNavigation = (evt, rowAndColumn, rowIndex, colIndex) => {
+  let nextRow = rowIndex;
+  let nextCol = colIndex;
+
+  switch (evt.key) {
+    case 'ArrowDown':
+      if (rowIndex + 1 < rowAndColumn.tableRowSize) nextRow = ++nextRow;
```

Comment:
```suggestion
      if (rowIndex + 1 < rowAndColumn.tableRowSize) nextRow++;
```

This is all you need to do, it is the same thing as `nextRow = nextRow + 1`

Reply:
right, gonna change. thanks.